pkg/cloudprovider/aws/fleet/capacity.go

```diff
@@ -34,74 +31,48 @@ type Capacity struct {
 	launchTemplateProvider *LaunchTemplateProvider
```

Comment:
I think we can remove `ec2iface.EC2API` field, since we moved this logic from here to `instanceProvider`